subsys/bluetooth/host/hci_core.c

```diff
@@ -1216,7 +1715,25 @@ static void update_pending_id(struct bt_keys *keys, void *data)
 		return;
 	}
 }
-#endif
+
+static inline void pending_id_keys_update_set(struct bt_keys *keys, u8_t flag)
+{
+	atomic_set_bit(bt_dev.flags, BT_DEV_ID_PENDING);
+	keys->flags |= flag;
+}
+
+static inline void pending_id_keys_update(void)
```

Comment:
What's the reason you've used inline for these? Usually that's only needed when defining functions in header files. In 99% of cases it's better to leave this up to the compiler to decide when it comes to c-files.

Reply:
Will remove.